Release_Protocol.md

```diff
@@ -95,8 +95,8 @@ Important note: The pull request title **must** be named "REL: vX.Y.Z" (for exam
 
 **This will open a period of discussion for 5 business days regarding if we are ready to release.**
 
-Minor revisions may be made using GitHub's [suggestion
-feature](https://help.github.com/en/articles/incorporating-feedback-in-your-pull-request).
+Minor revisions may be made using GitHub's
+[suggestion feature](https://help.github.com/en/articles/incorporating-feedback-in-your-pull-request).
```

Comment:
I reformatted several such "multiline links" to be on a single line. this is a style choice that makes it easier to parse the links ... and it's also nice on the eye (the latter just IMHO). I added a CI check for this, so this style can now be enforced.

Reply:
I think the advantage to this was to avoid a line break in GitHub's rendering of the document. It doesn't matter much to me, though.